scripts/frame/main.js

```diff
@@ -81,21 +81,21 @@ frame.next = function () {
 
 frame.refreshPicture = function () {
 	api.post("Photo::getRandom", {}, function (data) {
-		if (!data.url && !data.medium) {
+		if (!data.url && data.sizeVariants.medium === null) {
```

Comment:
before checking `data.sizeVariants.medium === null` you need to check that 
`data.sizeVariants !== null` Otherwise this code will throw an exception is `data.sizeVariants` is `null`. Yes, it is stupid I know. But better safe than sorry. :(

You can probably just do a check at the beginning if `sizeVariants` and `url` exists in order to avoid to propagate the checks down the line. ;)

Reply:
Fixed with commit 119e753